hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -351,7 +361,18 @@ private void handleUserTransaction(
                     readableStoreFactory,
                     fees,
                     platformEvent.getCreatorId().id());
+
+            networkUtilizationManager.resetFrom(state);
+
             if (validationResult.status() != SO_FAR_SO_GOOD) {
+                final var sigVerificationFailed = validationResult.responseCodeEnum() == INVALID_SIGNATURE;
+                if (sigVerificationFailed) {
+                    // If the signature status isn't ok, only work done will be fee charging
+                    // Note this is how it's implemented in mono (TopLevelTransition.java#L93), in future we may want to
+                    // not trackFeePayments() only for INVALID_SIGNATURE but for any preCheckResult.status() !=
```

Comment:
Can you please create a ticket to track this?

Reply:
Created https://github.com/hashgraph/hedera-services/issues/8874.